pages/data-providers/[data-provider-id]/deposit-compliance.jsx

```diff
@@ -105,13 +130,7 @@ const DepositCompliance = ({
         />
         <NumericValue
           tag="p"
-          value={valueOrDefault(
-            store.depositDates.complianceLevel &&
-              (100 - store.depositDates.complianceLevel).toFixed(2),
-            'Loading...'
-          )}
-          append="%"
-          caption="non-compliant"
+          {...getComplianceLevelNumberProps(store.depositDates.complianceLevel)}
```

Comment:
I do not think this should change. Maybe do not show `%` if the percentage is 0. It'd numeric value, so it's good to keep it numeric. I would not mentally expect to see _All_ instead the number.